src/chi/themes/centurylink/_variables.scss

```diff
@@ -6,6 +6,7 @@ $state-colors: (success: green, warning: yellow, danger: red);
 // Font family
 $font-family-base: 'Inter', Arial, Helvetica, Verdana, sans-serif;
 $font-family-mono: Menlo, Consolas, 'Liberation Mono', Courier, monospace;
+$font-family-icon: 'chi';
```

Comment:
@mattnickles @janibolkvadze is it neccessary to duplicate in each theme this variable when it is declared in global-variables.scss? Thoughts?

Reply:
I don't think it's necessary.

Reply:
I suggested storing this in themes as it will eventually make sense for themes to customize what icon font they would like to use (Chi, Font Awesome, Material, etc.). However, icon theming is not a current priority so please feel free to store in global.